api/serverpb/server.proto

```diff
@@ -137,6 +139,16 @@ message ChangeSettingsResponse {
   Settings settings = 1;
 }
 
+message UploadSSHKeyRequest {
```

Comment:
From my point of view `AddSSHKeyRequest` is a better fit as a name for this message.
The prefix "Upload" confusing a bit here.
An "Upload" gives some connotation that request has to have binary body and filename.
But we have here only string with a body of the key.

@BupycHuk, what do you think?

Reply:
The think about add is that we don't add ssh key, we replace ssh key. Maybe it's better to call it SetSSHKeyRequest.

Reply:
@BupycHuk, yes, it sounds even better.